parley/src/util.rs

```diff
@@ -14,3 +14,53 @@ pub(crate) fn nearly_eq(x: f32, y: f32) -> bool {
 pub(crate) fn nearly_zero(x: f32) -> bool {
     nearly_eq(x, 0.)
 }
+
+/// A rectangle.
+#[derive(Clone, Copy, Default, PartialEq)]
+pub struct Rect {
+    /// The minimum x coordinate (left edge).
+    pub x0: f64,
+    /// The minimum y coordinate (top edge in y-down spaces).
```

Comment:
Does Parley not already assume a y-down space? That is,

```suggestion
    /// The minimum y coordinate (top edge in Parley's y-down space).
```

Reply:
I have made these comments specific to Parley's coordinate space seeing as this type is not intended to be used elsewhere.